oci/pack.go

```diff
@@ -183,6 +183,8 @@ func NewPackageFromTarget(ctx context.Context, targ target.Target, opts ...packm
 		}
 	}
 
+	ocipack.command = append(ocipack.command, popts.Args()...)
```

Comment:
In this constructor method, `ocipack.command` is never initialized to a value so invoking `append` is redundant.

Perhaps it is better to move this bit of code:
```go
	popts := &packmanager.PackOptions{}
	for _, opt := range opts {
		opt(popts)
	}
```

Earlier such that it is before [the instantiation of `ocipack`](https://github.com/unikraft/kraftkit/blob/0433663540ad8d87b982d3c9b45694b017e07fc1/oci/pack.go#L60-L66) and so that `commands` can be set directly to `popts.Args()`.